kotlin/internal/jvm/compile.bzl

```diff
@@ -205,6 +205,10 @@ def _kotlinc_options_provider_to_flags(opts):
         flags.append("-Xallow-jvm-ir-dependencies")
     if opts.x_skip_prerelease_check:
         flags.append("-Xskip-prerelease-check")
+    if opts.include_stdlibs == "stdlib":
+        flags.append("-no-reflect")
```

Comment:
Doesn't reflect the include_stdlibs attribute.

Reply:
@restingbull mind elaborating?

Reply:
The `-no-reflect` flag strips out the Kotlin reflect library but keeps the standard library. The `-no-stdlib` flag strips both the reflect and standard libraries out. Not passing either of these flags will result in both the standard library and reflect being on the classpath.

When the `include_stdlibs ` is set to `all` we want to just include everything (pass no flags). When `include_stdlibs` is set to `stdlib` we want to pass -no-reflect which just strips the reflect standard libraries but keeps the kotlin standard library. Lastly when we pass `none` we want to just strip everything by passing `-no-stdlib`.

This logic also mirrors what @Kernald mentioned here https://github.com/bazelbuild/rules_kotlin/pull/395#issuecomment-728301116

Reply:
Nevermind, I'm going brain dead. I read the code as taking a list no less than 5 times. My apologies.